### PR TITLE
Fix squished thumbnails on list pages

### LINKED ISSUE #1762: Thumbnails get cut up/squished on list pages (on mobile) and in search results
Thumbnails are getting cut off on list pages.

In search autocomplete results......... and inside lists (e.g http://localhost:8080/account/books/want-to-read) they sometimes get squished.

screen shot 2018-12-26 at 2 55 53 pm

This seems to be caused by mixing height and width definitions such that the image is forced to be outside its aspect ratio

See screenshot below.

> Relevant url:

https://openlibrary.org/people/jdlrobson/lists

> Screenshot (if possible):
screen shot 2018-12-25 at 6 12 05 pm

The thumbnail should not be cut in half.

e.g. `https://openlibrary.org/...`

> Logged in (Y/N)?

> Browser type/version?

> Operating system?

### PR DESCRIPTION
**Pull request by koderjoker**
> **Description**: What does this PR achieve? [hotfix]

- Prevents thumbnails from getting squished in lists (mobile view)

- Thumbnails getting cut off resolved by PR #1838 by @jdlrobson 

Closes #1762 

> **Testing**: Steps for reviewer to reproduce / verify this PR fixes the problem?

http://0.0.0.0:8080/people/openlibrary/lists
http://0.0.0.0:8080/account/books/want-to-read

> **Evidence**: If this PR touches UI, please post evidence (screenshot) of it behaving correctly:

Lists (mobile view)

Want to read

Want to read (mobile view)

### CODE REVIEW

**Comment by koderjoker** [comment]
@jdlrobson made the necessary changes! The aspect ratio is preserved across all screens now. Will squash commits when changes are approved.

**Comment by mekarpeles** [comment]
testing now

File: static/css/components/search-result-item.less [hunk 1]
```diff
@@ -97,7 +105,28 @@
 }
 
 @media only screen and (min-width: @width-breakpoint-tablet) {
-  .searchResultItem {
-    flex-wrap: nowrap;
+  .list-books {
```
Comment (jdlrobson) [inline static/css/components/search-result-item.less]: Am not sure about this selector. 
It looks like you sre trying to increase specificity but I'd rather not do that if possible.

What happens when this is removed?

File: openlibrary/macros/SearchResultsWork.html [hunk 2]
```diff
@@ -13,7 +13,7 @@
         $ cover = "//archive.org/download/%s/page/cover_w60_h60.jpg" % doc.get('ocaid')
     $else:
         $ cover = "/images/icons/avatar_book-sm.png"
-    <a href="$book_url"><img itemprop="image" src="$cover" height="70" alt="Cover of: $doc.title$(': ' + doc.subtitle if doc.get('subtitle', None) else '')" title="Cover of: $doc.title$(': ' + doc.subtitle if doc.get('subtitle', None) else '')"/></a>
+    <a href="$book_url"><img itemprop="image" src="$cover" alt="Cover of: $doc.title$(': ' + doc.subtitle if doc.get('subtitle', None) else '')" title="Cover of: $doc.title$(': ' + doc.subtitle if doc.get('subtitle', None) else '')"/></a>
   </span>
```
Comment (jdlrobson) [inline openlibrary/macros/SearchResultsWork.html]: Ideally all img tags should have width and height attributes but also these shouldnt be fixed so I am okay with this removal

File: static/css/components/search-result-item.less [hunk 3]
```diff
@@ -97,7 +105,28 @@
...
+      .bookcover {
+        width: 15%;
+        text-align: right;
+      }
+      .details {
+        width: 55%;
+      }
+    }
+  }
+  .list-results {
```
Comment (jdlrobson) [inline static/css/components/search-result-item.less]: Same as above

**Comment by cdrini** [comment]
(side note: I haven't reviewed the above thoroughly, but justed wanted to quickly name-drop the css `object-fit` property in case you hadn't heard of it ( https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit ). Browser support is pretty great, and it makes dealing with these situations much easier! But that's just a side note...)

**Comment by mekarpeles** [comment]
I think we may need to do some followup/cleanup (the waitlist buttons have the wrong css applied, the cover images are too small in mobile view)

merging for now as I think the central issue is achieved

File: static/css/components/search-result-item.less (hunk 1 above)
Comment (koderjoker) [reply static/css/components/search-result-item.less]: Used selectors `list-books` and `list-results` to ensure the width of `details` remains different for pages lists and want-to-read

**Comment by koderjoker** [comment]
Thanks for merging this 😄

**Comment by jdlrobson** [comment]
Looks great on mobile! Nice job!
https://dev.openlibrary.org/account/books/want-to-read
https://dev.openlibrary.org/people/jdlrobson/lists

**Pull request merged**